### PR TITLE
add docraptor

### PR DESCRIPTION
**Pull request by mediocretes**

### CODE REVIEW

**Comment by deniskorobicyn** [comment]
I think the link should be https://github.com/expectedbehavior/doc_raptor_gem...

File: README.md [hunk 1]
```diff
@@ -677,6 +677,7 @@ Thanks to all [contributors](https://github.com/markets/awesome-ruby/graphs/cont
 
 ## PDF
 
+* [DocRaptor](https://github.com/expectedbehavior/doc_raptor) - For use with [DocRaptor](https://docraptor.com), convert html to pdf with ease and precision.
```
Comment (deniskorobicyn) [inline README.md]: I think the link should be https://github.com/expectedbehavior/doc_raptor_gem...

**Comment by markets** [comment]
Hi @mediocretes, not sure if it's a really well-known resource. Let's keep it open some time.

Thanks for sharing!

**Comment by jqr** [comment]
[Ruby Toolbox's PDF Generation](https://www.ruby-toolbox.com/categories/pdf_generation) page lists it as having more downloads than wisepdf, gimli, shrimp, and kitabu. It's pretty popular :)

```
rghost     98,157
doc_raptor 83,303
wisepdf    38,458
gimli      38,305
shrimp     14,634
kitabu     13,270
```

**Comment by markets** [comment]
This is right @jqr. But the gem it's a wrapper to the DocRaptor service, so it should be added under [Third-party APIs](https://github.com/markets/awesome-ruby#third-party-apis). And this category must be treated with care, it could grow a lot! I think we should just add wrappers to widely known/big services, ie: Salesforce, Facebook, Twitter, Gmail, HipChat, Slack, Youtube, GitHub, Instagram ... 

DocRaptor seems an awesome project! :+1:

**Comment by jqr** [comment]
I can see the reason for wanting that limitation, but it's an interesting case of not listing something many Rubyists find useful simply because it is a service. Also, DocRaptor is built entirely on Ruby :)

**Comment by markets** [comment]
Hi @jqr, sorry sometimes it's not easy to draw the line, but imagine if we start to add wrappers for all Ruby/Rails services/SaaS... IMHO that doesn't add value to AwesomeRuby.

I think you can submit DocRaptor to https://github.com/ekremkaraca/awesome-rails "A collection of awesome projects, sites made with Rails".

Btw, I'm not 100% sure and we always can consider exceptions :sweat_smile: (it really has a lot of downloads). Let's wait community's feedback (to finally merge it or no).

Thanks.

**Comment by markets** [comment]
As per this PR https://github.com/expectedbehavior/doc_raptor_gem/pull/18

> This gem is no longer maintained.

So, closing it for now.

**Pull request closed without merging**